pulp_rpm/app/viewsets.py

```diff
@@ -690,14 +683,18 @@ class CopyViewSet(viewsets.ViewSet):
     """
 
     serializer_class = CopySerializer
+    permission_classes = (CopyAccessPolicy,)
 
     DEFAULT_ACCESS_POLICY = {
         "statements": [
             {
                 "action": ["create"],
                 "principal": ["authenticated"],
                 "effect": "allow",
-                "condition": "has_model_perms:rpm.copy_rpmrepository",
+                "condition": [
+                    "has_perms_to_copy",
+                    "has_model_or_obj_perms:rpm.copy_rpmrepository",
```

Comment:
I don't think one can have object permission  `rpm.copy_rpmrepository` on the specific object.
https://github.com/pulp/pulpcore/blob/main/pulpcore/app/global_access_conditions.py#L66 what copyviewset would return here? :)

Reply:
To be honest I am questioning whether this permission is needed in general?

Reply:
You are right. One cannot have object permission to copy but the model permission. Question is if we want to protect the endpoint itself. E.g. user can't sync but can copy (of course if got at least one repo to write on). @dralley wdyt?